src/form/input/Input.tsx

```diff
@@ -77,10 +89,11 @@ function Input(props: InputProps) {
 
       <input
         className={inputClassName}
-        type={type}
+        type={isNumberInput ? "text" : type}
```

Comment:
As we discussed, NumberInput component should still be there but `Input` component should just render `NumberInput` if `type === "number"` and pass all of the props to the `NumberInput`

Reply:
Oh, I get it. `NumberInput` component should exist but using only by `Input` component. 👍🏻

Reply:
Just to give update for others, as NumberInput also uses Input, this wouldn't work and leads to an infinite loop. Therefore, after talking with Yasin, we decided that it would be better to remove NumberInput altogether and just keep Input component. We will do specific calculations within Input when `type` props is equal to "number".